services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -25,27 +24,70 @@ export const EntityRow = ({
   openDeleteEntityModal,
   entering,
   hasStandards,
+  hasPlans,
 }: EntityRowProps) => {
   const { isMobile } = useBreakpoint();
   const { report } = useStore();
   const { userIsEndUser } = useStore().user ?? {};
 
-  const { name, measure_name } = entity;
+  const {
+    report_programName,
+    report_planName,
+    name,
+    measure_identifierCbe,
+    measure_identifierCmit,
+    measure_name,
+  } = entity;
+  const reportingPeriod = `${entity.report_reportingPeriodStartDate} to ${entity.report_reportingPeriodEndDate}`;
   const { reportType } = report || {};
 
   const entityComplete = useMemo(() => {
     return getEntityStatus(entity, report, entityType);
   }, [report]);
 
+  const getMeasureIdDisplayText = () => {
+    let identifier = "N/A";
+    if (measure_identifierCmit) {
+      identifier = `CMIT: ${measure_identifierCmit}`;
+    } else if (measure_identifierCbe) {
+      identifier = `CBE: ${measure_identifierCbe}`;
+    }
+    return `Measure ID: ${identifier}`;
+  };
+
   const entityFields = () => {
+    let fields: string[] = [];
     switch (reportType) {
       case ReportType.MCPAR:
-        return [measure_name, getMeasureIdDisplayText(entity)];
+        fields = [measure_name, getMeasureIdDisplayText()];
+        break;
       case ReportType.MLR:
-        return getProgramInfo(entity);
+        fields = [
+          report_planName,
+          report_programName,
+          eligibilityGroup(entity),
+          reportingPeriod,
+        ];
+        break;
+      default:
+        fields = [name];
+    }
+    return fields;
+  };
+
+  const openDisabled = () => {
```

Comment:
now that we have multiple ways to disable (hasStandards and hasPlans), I think it'd make more sense to pass in a single prop for disabled rather than pass in multiple props and check them here. I can make another PR for that though

Reply:
opened a PR https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/12534